zenguard/zenguard.py

```diff
@@ -19,20 +19,14 @@
 )
 
 
-class SupportedLLMs:
```

Comment:
if we delete this, how does the user know what clients we support? He has to go and read implementation?